Resources/Prototypes/Entities/Structures/Walls/asteroid.yml

```
@@ -1272,6 +1316,28 @@
+  id: WallRockBasaltTungsten
+  parent: WallRockBasalt
+  description: An ore vein rich with wolframite.
+  suffix: Copper
+  components:
+    - type: OreVein
+      oreChance: 1.0
+      currentOre: OreTungsten
+    - type: Sprite
+      layers:
+        - state: rock_wall
+        - map: [ "enum.EdgeLayer.South" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)